metadata_service/config.py

```diff
@@ -77,7 +80,8 @@ class Config:
     PROGRAMMATIC_DESCRIPTIONS_EXCLUDE_FILTERS = []  # type: list
 
 
-class LocalConfig(Config):
+# NB: If you're using the gremlin proxy, the appropriate GremlinConfig must be added to any other configs
+class LocalConfig(LocalGremlinConfig, Config):
```

Comment:
is subclass GremlinConfig for testing purpose? If that's the case, could we do sth like https://github.com/amundsen-io/amundsenfrontendlibrary/blob/ccfd2d6b82957fef347e956b243e4048c191fc0d/amundsen_application/config.py#L136 which create a test config in the file?

Reply:
Ah, no, it inherits GremlinConfig so we don't need to litter the metadata service and databuilder config files with our gremlin endpoint information. The alternative is duplicating our gremlin-specific config in 3 different repos, which is unpleasant.